examples/lang_err_integration_tests/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "lang_err_integration_tests"
```

Comment:
Other examples are using `-` as a separator so we should be consistent

Reply:
But `cargo-contract` told me otherwise 😆 

```
❯ cargo contract new beep-boop
Contract names can only contain alphanumeric characters and underscores
```